CHANGELOG.md

```diff
@@ -0,0 +1,44 @@
+# ksh x.y.z (version TBD, this is a work in progress)
+
+Starting in May 2017 maintenance of the Korn shell (`ksh`) resumed with
```

Comment:
I received commit access at end of June. I started merging changes (that dates back before I got commit access) from my fork in July.